internal/entity/user.go

```diff
@@ -13,9 +13,19 @@ type User struct {
 
 // NewUser represent new user in request.
 type NewUser struct {
-	ID              string `json:"id" validate:"omitempty"`
-	Name            string `json:"name,omitempty" validate:"omitempty"`
-	Email           string `json:"email" validate:"required,email"`
+	ID    string `json:"id" validate:"omitempty"`
+	Name  string `json:"name,omitempty" validate:"omitempty"`
+	Email string `json:"email" validate:"required,email"`
+	PasswordsRequest
+}
+
+type PasswordsRequest struct {
+	UserID          string `json:"id" validate:"omitempty"`
```

Comment:
in user with ID  userID :smile: 
I need to do something with user entity, right now it's too much inside it, maybe i will move it to handlers.

Reply:
Heh, actually, I need only id from user there. I did not embed it.
I thought about Roman's idea. Possible solution is to store passwords separately

Reply:
not sure how to do it right, need some research and rewrite it couple of times and test

Reply:
I will do something like this https://threedots.tech/post/common-anti-patterns-in-go-web-applications/

Reply:
@igkostyuk As I understood, your idea is to use less embedding but to make several entities. Or you talk about dto and entities?